pom.xml

```
@@ -18,7 +18,7 @@
 		<dependency>
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-databind</artifactId>
-			<version>2.9.10.1</version>
+			<version>2.9.10.3</version>
```

bump to current version (increase minor)

```suggestion
			<version>2.10.3</version>
```